lib/microservices_router.rb

```diff
@@ -0,0 +1,12 @@
+require 'net/http'
+require 'uri'
```

Comment:
It's included in `Net::HTTP`, so @enragednuke is semi-correct. No need to require it here.